api/src/main/java/com/epam/pipeline/app/JWTSecurityConfiguration.java

```diff
@@ -142,7 +142,8 @@ protected String getSecuredResources() {
             "/launch.sh", "/PipelineCLI.tar.gz",
             "/pipe-common.tar.gz", "/commit-run-scripts/**", "/pipe",
             "/fsbrowser.tar.gz", "/error", "/error/**", "/pipe.zip", "/pipe.tar.gz",
-            "/pipe-el6", "/pipe-el6.tar.gz"
+            "/pipe-el6", "/pipe-el6.tar.gz",
```

Comment:
Could you please move this list to application.properties and use it in both security configurations

Reply:
added a new application property `static.common.scripts.exclude` which contains a list of paths to exclude